daffodil-runtime1/src/main/scala/org/apache/daffodil/runtime1/layers/api/Layer.java

```diff
@@ -63,11 +63,11 @@
  * <p/>
```

Comment:
THis doc needs updating to use the new setVariable method.

Reply:
updated in next commits